README.md

```diff
@@ -58,38 +58,77 @@ The following figure shows the typical CoatRack architecture, the CoatRack web a
 
 
 
-## How to Build CoatRack Container Images from Source
+## Usage
+
+To start using CoatRack: 
+
+- either run CoatRack locally as explained in the "Install ..." sections
```

Comment:
"Install" is misleading, see other comment below.

Reply:
Done as suggested in comment below.